pipeline-modules/deployment-service/gcp/cloud-run.yaml

```diff
@@ -87,11 +101,43 @@ template: |
     args:
     - '-c'
     - |
-      gcloud run deploy {{applicationName|lower}} --image {{repository}}:{{tag}} --region {{region}} \
-      --platform managed \
-      --labels codepipes-project={{project}} \
-      --labels codepipes-enviroment={{environment}} \
-      {% if allowUnauthenticated == true %} --allow-unauthenticated {% else %}  {% endif %}
+      echo "
+      apiVersion: serving.knative.dev/v1
+      kind: Service
+      metadata:
+        name: {{applicationName|lower}}
```

Comment:
Since we use this at couple of lines you could save it in a local variable and use that instead to ensure we use the lower-case name consistently.

`{% set appNameLowerCase = applicationName|lower %}`

Reply:
noted! https://cldcvr.atlassian.net/browse/VAN-2075